example/spring-data-mongodb/src/main/kotlin/com/github/inflab/example/spring/data/mongodb/repository/atlas/NearSearchRepository.kt

```diff
@@ -0,0 +1,149 @@
+package com.github.inflab.example.spring.data.mongodb.repository.atlas
+
+import com.github.inflab.example.spring.data.mongodb.annotation.Database
+import com.github.inflab.example.spring.data.mongodb.entity.airbnb.ListingsAndReviews
+import com.github.inflab.example.spring.data.mongodb.entity.airbnb.ListingsAndReviewsAddress
+import com.github.inflab.spring.data.mongodb.core.aggregation.aggregation
+import com.github.inflab.spring.data.mongodb.core.mapping.rangeTo
+import org.springframework.data.mongodb.core.MongoTemplate
+import org.springframework.data.mongodb.core.aggregation.AggregationResults
+import org.springframework.data.mongodb.core.geo.GeoJsonPoint
+import org.springframework.data.mongodb.core.mapping.Field
+import org.springframework.stereotype.Repository
+import java.time.LocalDateTime
+
+@Repository
+class NearSearchRepository(
+    @Database("sample_mflix") private val mflixMongoTemplate: MongoTemplate,
+    @Database("sample_airbnb") private val airbnbMongoTamplate: MongoTemplate,
+) {
+
+    data class RuntimeDto(
+        val title: String,
+        val runtime: Int,
+        val score: Double,
+    )
+
+    data class ReleasedDto(
+        val title: String,
+        val released: LocalDateTime,
+        val score: Double,
+    )
+
+    data class GeoDto(
+        val name: String,
+        val address: ListingsAndReviewsAddress,
+        val score: Double,
+    )
+
+    data class GeoPropertyTypeDto(
+        @Field("property_type")
+        val propertyType: String,
+        val address: ListingsAndReviewsAddress,
+        val score: Double,
+    )
+
+    fun findByRuntime(): AggregationResults<RuntimeDto> {
+        val aggregation = aggregation {
+            search {
+                index = "runtimes"
+                near {
+                    path("runtime")
```

Comment:
Since we have collection of movie document,
We can use `KProperty` instead of `String`.
I think it would be better to guide users to use ﻿KProperty instead of strings if possible.
(it applies `project` stage too)

```suggestion
                    path(Movies::runtime)
```